### ISSUE TITLE
Packages: monitoring readiness for GAP 4.12

### ISSUE THREAD

**Issue by alex-konovalov** [gap-system/gap#3995]
This is an issue to monitor the status of selected packages for which an update is desired in time for GAP 4.12. This replaces #2984 and lists everything from there which was not ready by the time of GAP 4.11. Of course, some of these issues may be addressed in package releases included in GAP 4.10 series. Please feel free to edit this issue and add more packages to the checklist. 

**HIGHLY DESIRABLE**
- [x] GBNP

**GOOD TO HAVE**
- [ ] More fixes to make tests pass for packages from <s>[staging tests](https://travis-ci.org/gap-system/gap-docker-pkg-tests-stable-4.10-staging)</s> [PackageDistro tests](https://gap-system.github.io/PackageDistro/latest-master/redirect.html):
  - [x] atlasrep: diffs reported and fixes suggested in #1617
  - [x] FR to fix #3418
  - [x] gbnp: @jwknopper contacted regarding the new release (including new repository and webpage)
  - [x] hapcryst: reported to the author by @alex-konovalov in October 2018 (CC GAP Support)
  - [x] polymaking: check if it is usable, and can it be  superseded by https://github.com/homalg-project/homalg_project/tree/master/PolymakeInterface ?
  - [x] fining: make a new release (-> @fingolfin)
  - [x] forms: make a new release (-> @fingolfin)
  - [x] io: make a new release (-> @fingolfin)
  - [ ] polycyclic: badly needs some fixes and a new release (-> @fingolfin)
- [ ] some packages currently submitted for the refereeing 

**PULL REQUESTS BLOCKED BY PACKAGES**
- [ ] #3006 waits for Semigroups, GUAVA. Done in numericalspgs.
- [ ] #2997 waits for NormalizInterface. 

**WINDOWS ISSUES**
Need also to check that as many as possible packages with binaries are usable under Windows.
- [ ] JupyterKernel https://github.com/gap-packages/JupyterKernel/issues/45

**ESTABLISHING PUBLIC REPOSITORIES**

The list of remaining packages which are redistributed with GAP but do not have known (to us) public repository is at https://github.com/gap-packages/gap-packages.github.io/issues/7. Please help to reduce it - by letting us know if the repository exists, of if you're a package author, by establishing it. We offer help with this:
- we can prepare for you a repository with revisions history populated from the past releases of your package
- we can automate publishing releases for you, and 
- we can run tests of the **development** version of your package, not only of its latest release.

**Comment by fingolfin** [comment]
I am satisfied with the readiness of packages.
